src/mocks/supabaseData/publicProjects.ts

```diff
@@ -123,33 +150,42 @@ export const fakeProjects = [
       "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
     location: "Berlin",
     devices: [{ records: [] }],
+    user: {
+      name: "Dennis",
+    },
   },
   {
     id: 12,
     name: "Project 3",
     description:
       "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-    location: "Berlin",
+    location: "Lucas",
```

Comment:
@vogelino wieso gibst dem location einen menschen namen? Ich dachte das soll ein ort name sein eg Berlin

Reply:
Right! My bad. I misplaced this value. I will change that!